packages/admin/README.md

```diff
@@ -0,0 +1,20 @@
+# @textile/hub-admin
+
+**Info**
+
+This is not a supported developer library.
```

Comment:
hoping this is enough to dissuade would be installers from using this :P

Reply:
maybe the inverse

Reply:
Example?

Reply:
oh, ha, i just meant it may encourage use :D

Reply:
lol, I see! yes that's entirely possible :P